src/main/java/uk/gov/justice/digital/delius/data/api/OffenderRecalledNotification.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class OffenderRecalledNotification {
+    @ApiModelProperty(value = "The date the recall occurred", example = "2020-10-25")
```

Comment:
very minor but important, but I think this should be "The date the person was returned to custody"

Reply:
Sure.  will update